tests/integration/ha_tests/test_smoke.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+import os
+from asyncio import TimeoutError
+
+import pytest
+from pytest_operator.plugin import OpsTest
+from tenacity import Retrying, stop_after_delay, wait_fixed
+
+from ..helpers import (
+    CHARM_SERIES,
+    DATABASE_APP_NAME,
+    scale_application,
+)
+from .helpers import (
+    apply_pvc_config,
+    change_pv_reclaim_policy,
+    change_pvc_pv_name,
+    check_db,
+    check_system_id_mismatch,
+    create_db,
+    delete_pvc,
+    get_any_deatached_storage,
+    get_pv,
+    get_pvc,
+    get_storage_id,
+    is_postgresql_ready,
+    is_storage_exists,
+    remove_pv_claimref,
+    remove_unit_force,
+)
+
+TEST_DATABASE_RELATION_NAME = "test_database"
+DUP_DATABASE_APP_NAME = DATABASE_APP_NAME + "2"
+
+logger = logging.getLogger(__name__)
+
+env = os.environ
+env["KUBECONFIG"] = os.path.expanduser("~/.kube/config")
+print(f"Model Name: {DATABASE_APP_NAME}")
```

Comment:
This `print` statement may be removed. I think it was used only for testing.

Reply:
fixed